javascript/javascript1/week2/flight-booking.js

```diff
@@ -0,0 +1,34 @@
+//Flight booking
+function getFullName(firstName, surName){
+    return firstName+' '+surName;
+}
+let fullName1=getFullName("Benjamin", "Hughes");
+let fullName2=getFullName("Iza", "David");
+console.log(fullName1);
+console.log(fullName2);
+
+//formal fullname
+function getFormalFullname(firstName, surName,useFormalName=true,gender){
+    if((firstName && surName)==='')
```

Comment:
`(firstName && surName)===''
`
JavaScript would evaluate this expression as follow

1. (firstName && surName) -> true or false (boolean value)
2.  true[false] === ' ' - > this is always false.
I believe your intention is not to have this expression to evaluate `false` all the time

Reply:
I changed it to (firstName==='' || surName==='')